.circleci/config.yml

```diff
@@ -20,6 +20,7 @@ jobs:
             - .bundle
       - restore_cache:
           key: v1-pods-{{ checksum "Podfile.lock" }}
+      - run: brew install swiftlint
```

Comment:
I think this might also need to go into the README for requirements :)

Reply:
Good point, I have opted in by using swiftlint via Cocoapods so it does not introduce additional steps.